.github/workflows/release.yml

```diff
@@ -18,5 +18,7 @@ jobs:
       language: flutter
       language-version: '3.x'
       appId: ${{ vars.APP_ID }}
+      semantic-release-extra-plugins: '@fingerprintjs/semantic-release-native-dependency-plugin@^1.0.0'
+      prepare-command: 'echo "flutter.sdk=$FLUTTER_ROOT" > $GITHUB_WORKSPACE/android/local.properties'
```

Comment:
why was this needed? why did we need to add it now, and not needed it before?

Reply:
Because `release.yml` never triggered Android build but now `semantic-release-native-dependency-plugin` triggering `gradle` command. Btw you can check the commit message for this information.

Reply:
I see that we are not doing this for React Native. Any idea why it is not needed there but needed here?

Reply:
Do you mean `apply` change or this? React Native doesn't expect the `flutter.sdk` path and didn't implement the `dev.flutter.flutter-plugin-loader` gradle plugin. React Native project have `package.json`, so you don't need to define `semantic-release-extra-plugins` to install the plugin.

Reply:
I was asking for Flutter SDK path. Got it, thanks